Dockerfile

```diff
@@ -1,21 +1,58 @@
+# syntax=docker/dockerfile:experimental
+
+# Build stage: Install python dependencies
+# ===
+FROM ubuntu:bionic AS python-dependencies
+RUN apt-get update && apt-get install --no-install-recommends --yes python3-pip python3-setuptools
+ADD requirements.txt /tmp/requirements.txt
+RUN --mount=type=cache,target=/root/.cache/pip pip3 install --user --requirement /tmp/requirements.txt
+
+
+# Build stage: Install yarn dependencies
+# ===
+FROM node:10-slim AS yarn-dependencies
+WORKDIR /srv
+ADD package.json .
+RUN --mount=type=cache,target=/usr/local/share/.cache/yarn yarn install
+
+# Build stage: Run "yarn run build-js"
+# ===
+FROM yarn-dependencies AS build-js
+ADD static/js static/js
+ADD webpack.config.js .
+RUN yarn run build-js
+
+# Build stage: Run "yarn run build-css"
+# ===
+FROM yarn-dependencies AS build-css
+ADD static/sass static/sass
+RUN yarn run build-css
+
+# Build the production image
+# ===
 FROM ubuntu:bionic
 
+ADD . .
+# Install python and import python dependencies
+RUN apt-get update && apt-get install --no-install-recommends --yes python3-lib2to3 python3-pkg-resources ca-certificates libsodium-dev
+COPY --from=python-dependencies /root/.local/lib/python3.6/site-packages /root/.local/lib/python3.6/site-packages
+COPY --from=python-dependencies /root/.local/bin /root/.local/bin
+ENV PATH="/root/.local/bin:${PATH}"
+
 # Set up environment
 ENV LANG C.UTF-8
+ENV SECRET_KEY="secret_key"
```

Comment:
I am not sure we should add this in the Dockerfile itself. Maybe this should be added as a parameter when running the project with Docker: `docker run -ti -p "8006:80" -e SECRET_KEY="test"
anbox-cloud.io`

Since the env var is added in prod as a secret k8s should handle that.

Reply:
Do we need to add this to the `./run` then?

Reply:
No it's not needed. This is for the production image. The `./run` should read from the `.env` and the `.env.local` directly